doc/user/content/get-started.md

```diff
@@ -295,9 +295,9 @@ In streaming, time keeps ticking along — unlike in batch, where data is frozen
 That's it! You just created your first transformations on streaming data using common SQL patterns, and tried to break Materialize! We encourage you to continue exploring the sample Kafka source, or try one of the use-case specific quickstarts. If you’re done for the time being, remember to clean up the environment:
 
 ```sql
-DROP CLUSTER quickstart CASCADE;
-
 DROP SCHEMA qck CASCADE;
+
+DROP CLUSTER quickstart;
```

Comment:
Should we add the reset cluster command to avoid an error in the following command?
```suggestion
DROP CLUSTER quickstart;
RESET CLUSTER;
```

Reply:
+1. I didn't know the `RESET` command was a thing!

Reply:
Added for both `search_path` and `cluster`.